scripts/50_contract-admin.js

```diff
@@ -391,30 +390,24 @@ module.exports = async function contractAdmin() {
             const tlRequestParams = [
                 proxyAdmin.address,
                 0,
-                txData.value,
+                contractDeployInfo.upgradeData,
                 '0x0000000000000000000000000000000000000000000000000000000000000000',
                 '0x0000000000000000000000000000000000000000000000000000000000000000',
             ];
-            const timelockContractTxRequestToExecute = await timelockContract
-                .connect(deployer)
-                .populateTransaction['execute'](...tlRequestParams);
-            console.log(
-                'timelockContractTxRequestToExecute',
-                timelockContractTxRequestToExecute
-            );
 
-            const proxyAdminOwner = await proxyAdmin.owner();
-            await safeContractProposal(
-                timelockContractTxRequestToExecute,
-                deployer,
-                dlcAdminSafes.critical
-            );
+            console.log('Executing upgrade...');
+            await timelockContract
+                .connect(deployer)
+                .execute(...tlRequestParams);
 
             console.log('Updating DeploymentInfo...');
             await fs.copyFile(
                 `deploymentFiles/${network}/${contractName}.proposed.json`,
                 `deploymentFiles/${network}/${contractName}.json`
             );
+            await fs.rm(
+                `deploymentFiles/${network}/${contractName}.proposed.json`
+            );
```

Comment:
perhaps better to not delete this, in case anything goes wrong and/or we want to review it for historical reasons. Maybe it could be saved with a unique name, like with a date extension, so that we can keep it forever?